src/components/calendarfunc.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from "react";
+import Calendar from "react-calendar";
+import "./Calendar.css";
+import Checklist from "./checklist";
+import MemoFunc from "./memo";
+
+import Dropdown from "react-bootstrap/Dropdown";
+
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+const CalendarFunc = () => {
```

Comment:
Is this a function of a component? I would just name this `Calendar`, just like the css file